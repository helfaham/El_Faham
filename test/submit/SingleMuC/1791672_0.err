WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
condor_exec.exe: line 19: cd: Haamm/HaNaMiniAnalyzer/test: No such file or directory
----- Begin Fatal Exception 25-Mar-2019 10:58:41 CET-----------------------
An exception of category 'ConfigFileReadError' occurred while
   [0] Processing the python configuration file named PUAnalyzer_cfg.py
Exception Message:
python encountered the error: <type 'exceptions.IOError'>
[Errno 2] No such file or directory: 'PUAnalyzer_cfg.py'
----- End Fatal Exception -------------------------------------------------
ls: cannot access out*SingleMuC*.root: No such file or directory
ls: cannot access *out*SingleMuC*.root: No such file or directory
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
cp: missing destination file operand after `/eos/home-h/helfaham//'
Try `cp --help' for more information.
