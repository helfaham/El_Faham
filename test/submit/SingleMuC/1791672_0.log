000 (1791672.000.000) 03/25 10:28:34 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&noUDP&sock=2690491_9e79_37>
...
001 (1791672.000.000) 03/25 10:58:26 Job executing on host: <188.184.132.155:9618?addrs=188.184.132.155-9618+[--1]-9618&noUDP&sock=9513_a5c0_3>
...
006 (1791672.000.000) 03/25 10:58:35 Image size of job updated: 11828
	12  -  MemoryUsage of job (MB)
	11820  -  ResidentSetSize of job (KB)
...
005 (1791672.000.000) 03/25 11:03:28 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 00:00:01, Sys 0 00:00:01  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:01, Sys 0 00:00:01  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	11794  -  Run Bytes Sent By Job
	3609  -  Run Bytes Received By Job
	11794  -  Total Bytes Sent By Job
	3609  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :      489        4   1635975 
	   Memory (MB)          :       12     2000      2000 
...
