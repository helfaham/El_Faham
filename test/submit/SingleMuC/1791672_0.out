subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=helfaham/CN=834699/CN=Hesham El Faham/CN=1975496168
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=helfaham/CN=834699/CN=Hesham El Faham
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=helfaham/CN=834699/CN=Hesham El Faham
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/h/helfaham/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/test/submit/.x509up_u119445
timeleft  : 999:29:31
key usage : Digital Signature, Key Encipherment
0
cmsRun PUAnalyzer_cfg.py sample=SingleMuC job=0 output=out maxEvents=-1 nFilesPerJob=2
total 0
1th Try
2th Try
3th Try
4th Try
5th Try
6th Try
7th Try
8th Try
9th Try
10th Try
11th Try
12th Try
13th Try
14th Try
15th Try
16th Try
17th Try
18th Try
19th Try
20th Try
21th Try
The file was not copied to destination after 20 tries
